windows/drawtext.cpp

```diff
@@ -534,3 +534,47 @@ void uiDrawTextLayoutExtents(uiDrawTextLayout *tl, double *width, double *height
 	// TODO make sure the behavior of this on empty strings is the same on all platforms (ideally should be 0-width, line height-height; TODO note this in the docs too)
 	*height = metrics.height;
 }
+
+void uiDrawLoadDefaultFont(uiFontDescriptor *f)
+{
+	fontCollection *collection;
+	IDWriteGdiInterop *gdi;
+	IDWriteFont *dwfont;
+	IDWriteFontFamily *dwfamily;
+	NONCLIENTMETRICS metrics;
```

Comment:
Consistency nit: Use `NONCLIENTMETRICSW` and `SystemParametersInfoW()` directly.